WcaOnRails/app/controllers/application_controller.rb

```diff
@@ -73,14 +73,12 @@ def doorkeeper_unauthorized_render_options(error: nil)
     super
   end
 
-  private def redirect_unless_user(action, *args)
-    redirected = false
-    unless current_user && current_user.send(action, *args)
+  private def redirect_to_root_unless_user(action, *args)
+    redirected = !current_user&.send(action, *args)
```

Comment:
It bugged me too, but not enough to think about changing it until you mentioned it.

It's now `redirecting`.